cpp/cugraph_module/algorithms/balanced_cut_clustering.cu

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "mg_cugraph_utility.hpp"
+
+namespace {
+// TODO: Check Balanced Cut API
```

Comment:
Is this done?

Reply:
It is, however TODO  will stay till Legacy is changed in cuGraph repository

Reply:
Maybe change just TODO: update once new API is out